src/EventHub/EventHub/help/Add-AzEventHubIPRule.md

```diff
@@ -0,0 +1,180 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.EventHub.dll-Help.xml
+Module Name: Az.EventHub
+online version: https://docs.microsoft.com/en-us/powershell/module/az.eventhub/add-azeventhubiprule
+schema: 2.0.0
+---
+
+# Add-AzEventHubIPRule
+
+## SYNOPSIS
+Add a single IPrule to the NetworkRuleSet of the given Namespace
+
+## SYNTAX
+
+### IPRulePropertiesParameterSet (Default)
+```
+Add-AzEventHubIPRule [-ResourceGroupName] <String> [-Name] <String> [-IpMask] <String> [-Action <String>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### IPRuleInputObjectParameterSet
+```
+Add-AzEventHubIPRule [-ResourceGroupName] <String> [-Name] <String>
+ [-IpRuleObject] <PSNWRuleSetIpRulesAttributes> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Add a single IPrule to the NetworkRuleSet of the given Namespace
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Add-AzEventHubIPRule -ResourceGroupName v-ajnavtest -Namespace Eventhub-Namespace1-2389 -IpMask "11.22.33.44" -Action Allow
```

Comment:
@v-Ajnava all examples should have the corresponding output from the command execution

This comment applies to all markdown files in this PR